include/security.php

```diff
@@ -261,9 +268,9 @@ function permissions_sql($owner_id,$remote_verified = false,$groups = null) {
 	 * done this and passed the groups into this function.
 	 */
 
-	elseif($remote_user) {
+	elseif ($remote_user) {
```

Comment:
This statement is super awkward on its own, please move the above comment inside the `elseif` brackets and connect the `elseif` with the above `if` block as it should be.